app/page.tsx

```diff
@@ -41,6 +41,15 @@ function compareFn(a: any, b: any) {
   return 0;
 }
 
+function compareSurfacesFn(a: any, b: any) {
```

Comment:
```suggestion
function compareSurfacesFn(a: FxMSMessageInfo, b: FxMSMessageInfo): number {
```

As part of our ongoing cleanup of page.tsx, how about moving this to `messageUtil.ts`?